proxy/hdrs/MIME.h

```diff
@@ -852,6 +872,23 @@ MIMEField::value_append(HdrHeap *heap, MIMEHdrImpl *mh, const char *value, int l
   mime_field_value_append(heap, mh, this, value, length, prepend_comma, separator);
 }
 
+/*-------------------------------------------------------------------------
+  -------------------------------------------------------------------------*/
+
+inline bool
+MIMEField::value_validate() const
```

Comment:
It should be called value_is_valid, going with the normal naming convention of methods that return bool.